repository/t.json

```diff
@@ -3761,6 +3761,18 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"name": "Typst",
+			"details": "https://github.com/hyrious/typst-syntax-highlight",
+			"author": "hyrious",
+			"labels": ["tex", "typst"],
```

Comment:
The common label for syntax, highlighting, language support, etc, packages is "language syntax".

Reply:
Ok.